


class NotFoundException(Exception):
    pass

class DuplicatedException(Exception):
    pass